packages/cms-lib/hubdb.js

```diff
@@ -50,20 +49,40 @@ async function createHubDbTable(portalId, src) {
 
   let response;
   if (rowsToUpdate.length > 0) {
-    response = await createRows(portalId, id, rowsToUpdate);
+    response = await createRows(portalId, tableId, rowsToUpdate);
   }
 
-  await publishTable(portalId, id);
+  await publishTable(portalId, tableId);
 
   return {
-    tableId: id,
+    tableId,
     rowCount:
       response && Array.isArray(response) && response.length
         ? response[0].rows.length
         : 0,
   };
 }
 
+async function createHubDbTable(portalId, src) {
+  validateJsonFile(src);
+
+  const table = fs.readJsonSync(src);
+  const { rows, ...schema } = table;
+  const { columns, id } = await createTable(portalId, schema);
+
+  await updateRows(portalId, id, rows, columns);
+}
+
+async function updateHubDbTable(portalId, tableId, src) {
```

Comment:
@gcorne the idea behind this is that it would:
- Update the table meta/columns
- Add/append any rows that get passed to it

In the UI, you can either `append` or `replace` your rows. This solves for appending, but you'd need to couple this with `clearHubDbTableRows` in order to replace

Reply:
I think we should split the updating of the table from the adding of the rows, so remove line 83 and return the updated table and then in the code using these new functions add the rows separately, which will require exporting `addRows`.